recipes/perl-test-yaml/meta.yaml

```diff
@@ -8,20 +8,18 @@ source:
   md5: 92e6ea57576428095fe3c9b8e00e6f29
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
-    - perl-threaded
+    - perl
     - perl-test-base
 
   run:
-    - perl-threaded
+    - perl
     - perl-test-base
 
 test:
-  # Perl 'use' tests trigger "readline() on unopened filehandle DATA" errors,
-  # so we have to use this "dummy" test package instead.
```

Comment:
I suppose this is a useful comment as it explains the reason for the "non-standard" test case. I'd vote to keep it as long as that workaround is needed.